docs/devices_api.yml

```diff
@@ -0,0 +1,127 @@
+openapi: 3.0.3
+
+info:
+  title: Device Connect
+  description: |
+    Device facing API for managing persistent device connections.
+
+  version: "1"
+
+  contact:
+    name: Mender Support
+    email: support@mender.io
+    url: https://mender.io/contact-us
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: https://hosted.mender.io/api/devices/v1/deviceconnect
+
+# Global security definitions
+security:
+  - DeviceJWT: []
+
+# Global tags
+tags:
+  - name: DevicesAPI
+
+paths:
+
+  /connect:
+    get:
+      operationId: Connect
+      summary: >
+        Connect the device and make it available to the server.
+      description: >
+        Calling /connect will upgrade the connection to a persistent
+        websocket connection and making the device available to the
+        management api. The device must provide it's identity using
```

Comment:
`APIs`? I think we usually refer to them in plural.

Reply:
I think singular "API" is correct in this context as I'm also being particular with "management API". But +1 or uppercase abbreviation.